Lib/fractions.py

```diff
@@ -216,6 +216,14 @@ def from_decimal(cls, dec):
                 (cls.__name__, dec, type(dec).__name__))
         return cls(*dec.as_integer_ratio())
 
+    def as_integer_ratio(self):
+        """Return integer ratio.
+
+        Return a pair of integers, whose ratio is exactly equal to the original
```

Comment:
Would it be potentially helpful to readers to specify in the doc-string that the integer pair is returned as a tuple? This is not uncommonly done, but I think it's worthwhile to specifically mention this.
```suggestion
        Return a tuple containing two integers, whose ratio is exactly equal to the original
```

Reply:
Will do.